File: test/fixtures/kotlin/get_with_header_lowercase_cookie.kt
import java.io.IOException
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val request = Request.Builder()
  .url("http://localhost:28139/page")
  .header("cookie", "secure_user_id=InNlY3VyZTEwNjI3Ig%3D%3D--3b5df49345735791f2b80eddafb630cdcba76a1d; adaptive_image=1440; has_js=1; ccShowCookieIcon=no; _web_session=Y2h...e5")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}

File: test/fixtures/r/get_with_header_lowercase_cookie.r
library(httr)

cookies = c(
  secure_user_id = "InNlY3VyZTEwNjI3Ig==--3b5df49345735791f2b80eddafb630cdcba76a1d",
  adaptive_image = "1440",
  has_js = "1",
  ccShowCookieIcon = "no",
  `_web_session` = "Y2h...e5"
)

res <- httr::GET(url = "http://localhost:28139/page", httr::set_cookies(.cookies = cookies))
